titus-server-federation/src/main/java/com/netflix/titus/federation/service/AggregatingJobManagementService.java

```diff
@@ -164,14 +174,25 @@ public Completable updateJobStatus(JobStatusUpdate request) {
                     .setPagination(emptyGrpcPagination(request.getPage()))
                     .build());
         }
+
+        Set<String> fieldsFilter = Collections.emptySet();
+        if (request.getFieldsCount() > 0) {
+            fieldsFilter = new HashSet<>(request.getFieldsList());
+            fieldsFilter.addAll(JOB_MINIMUM_FIELD_SET);
```

Comment:
Would missing this have been added at Gateway or Master previously or would it have failed validation?

Reply:
master adds it too by default